frontend/src/components/Dialogs/InputDialog.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div v-if="dialog.state.active" class="modal-background">
-    <div class="modal-container">
+    <div :class="['modal-container', dialog.state.warningText ? 'limit-width' : '']">
```

Comment:
If I'm understanding this correctly, the modal container, without text, has a `min-width` of 25 rem. Then this does a v-if check to see if there is any text in the modal and then applies an additional class to the container to have a `max-width` of 25 rem.

This becomes a `min-width` of 25 rem and a `max-width` of 25 rem. Which means that the modal component is constrained to 25 rem totally.

Eliminate the check and make the modal's `width` 25 rem.

Reply:
If I were to do this, it would cause an issue with the other input dialog models that use the comment box that can be expanded beyond the width of the modal container. In the design for this feature, it was requested that the modal stay the size that it typically is. That is why this code exists.

Example with it as it is:
![image](https://github.com/uab-cgds-worthey/rosalution/assets/4248757/29ea75bb-1eca-433c-8c53-70323ff33079)


Example with your suggestions:
![image](https://github.com/uab-cgds-worthey/rosalution/assets/4248757/ee43bf2b-d1da-429c-8992-344aae54fba0)


We could limit the comments to only being allowed to be expanded in height, but I think we would need further clarification on that and possibly make a design decision. How would you like me to proceed?

Reply:
Given they warrant further design discussions, I would like you to proceed by documenting this occurrence in the relevant ticket as they don't get lost.

Once this is done, I'll resolve the conversation again.

Reply:
documented in the ticket